src/main/resources/assets/integrateddynamics/lang/en_US.lang

```diff
@@ -807,6 +809,95 @@ operator.operators.integrateddynamics.cast.name=Cast
 operator.operators.integrateddynamics.cast.basename=Cast
 operator.operators.integrateddynamics.cast.tooltip=Cast %s to %s
 
+## Config categories
+config.integrateddynamics.item=Item settings
+config.integrateddynamics.item.tooltip=Change settings of items
+config.integrateddynamics.block=Block settings
+config.integrateddynamics.block.tooltip=Change block settings
+config.integrateddynamics.fluid=Fluid settings
+config.integrateddynamics.fluid.tooltip=Change fluid settings
+config.integrateddynamics.entity=Entity settings
+config.integrateddynamics.entity.tooltip=Change entity settings
+config.integrateddynamics.general=General settings
+config.integrateddynamics.general.tooltip=Change general mod settings
+config.integrateddynamics.worldgeneration=World generation settings
+config.integrateddynamics.worldgeneration.tooltip=Change world generation settings
+config.integrateddynamics.enchantment=Enchantment settings
+config.integrateddynamics.enchantment.tooltip=Change enchantment settings
+config.integrateddynamics.mob=Mob settings
+config.integrateddynamics.mob.tooltip=Change mob settings
+config.integrateddynamics.biome=Biome generation settings
+config.integrateddynamics.biome.tooltip=Change biome generation settings
+config.integrateddynamics.machine=Machine settings
+config.integrateddynamics.machine.tooltip=Change machine settings
+config.integrateddynamics.potion=Potion settings
+config.integrateddynamics.potion.tooltip=Change potion settings
+config.integrateddynamics.core=Core settings
+config.integrateddynamics.core.tooltip=Change core settings
+config.integrateddynamics.modcompat=Mod compatibility settings
+config.integrateddynamics.modcompat.tooltip=Change mod compatibility settings
+config.integrateddynamics.capability=Capability settings
+config.integrateddynamics.capability.tooltip=Change capability settings
+
+# Configuration
+config.integrateddynamics=%s configuration
+
+## Biome settings
+config.integrateddynamics.biomeMeneglin=Enable Meneglin biome
+config.integrateddynamics.biomeMeneglin.spawnWeight=Meneglin biome Spawn Weight
+
+## Block settings
+config.integrateddynamics.menrilLeaves.berriesDropChance=Berries drop chance
+
+## Capability settings
+capability.integrateddynamics.cableFakeable=Allow fake cables
```

Comment:
I would opt for removing "Allow" and "Enable", so that the name just becomes "Fake cables" or "Dynamic light" for example.